packages/test-commons/src/generateIsomorphicTests.ts

```diff
@@ -54,6 +58,21 @@ export function generateIsomorphicTests(
                         // errors will fail the test and log full stack traces to the console. nifty!
                         Enzyme.render(element);
                     });
+                    it(`<${componentName}> className is applied to outer element`, () => {
```

Comment:
this test name is not always accurate... might want to do the `classNameChildList` check outside this `it()` so we can change the test name?